src/test/resources/org/kohsuke/github/AppTest/wiremock/blob/mappings/repos_github-api_github-api-2-db8eac.json

```diff
@@ -3,7 +3,12 @@
   "name": "repos_github-api_github-api",
   "request": {
     "url": "/repos/github-api/github-api",
-    "method": "GET"
+    "method": "GET",
+    "headers": {
+      "Accept": {
+        "equalTo": "text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2"
```

Comment:
It sucks to have to add this to almost every test mapping file, but it will be done automatically in the future.